core/transport/src/grpc/job_master.proto

```diff
@@ -142,7 +150,7 @@ service JobMasterClientService {
 message JobHeartbeatPOptions {}
 message JobHeartbeatPRequest {
   optional int64 workerId = 1;
-  repeated TaskInfo taskInfos = 2;
+  repeated JobInfo taskInfos = 2;
```

Comment:
Should this be called `jobInfos` instead?

Reply:
Nope this one should actually be taskInfos.

Reply:
The thing about JobHeartbeat is that they are really TaskHeartbeat from TaskWorkers (which will also be needing a rename)